components/TopBarProfileMenu.js

```diff
@@ -383,7 +383,7 @@ class TopBarProfileMenu extends React.Component {
     const { LoggedInUser } = this.props;
 
     return (
-      <StyledProfileButton isBorderless onClick={this.toggleProfileMenu}>
+      <StyledProfileButton isBorderless textAlign="left" onClick={this.toggleProfileMenu}>
```

Comment:
I missed that part during my first review. Using a `button` as the top container is a bit risky:
- Cursor is wrong: you get a pointer while not hovering anything
- DOM nesting: see the console:

```
next-dev.js?3515:25 Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>.
    at button
    at O (webpack-internal:///./node_modules/styled-components/dist/styled-components.browser.esm.js:31:19811)
    at _c2 (webpack-internal:///./components/StyledButton.js:57:22)
    at O (webpack-internal:///./node_modules/styled-components/dist/styled-components.browser.esm.js:31:19811)
    at div
    at O (webpack-internal:///./node_modules/styled-components/dist/styled-components.browser.esm.js:31:19811)
    at Reference (webpack-internal:///./node_modules/react-popper/lib/esm/Reference.js:16:23)
    at Manager (webpack-internal:///./node_modules/react-popper/lib/esm/Manager.js:13:23)
    at StyledTooltip (webpack-internal:///./components/StyledTooltip.js:153:183)
    at div
    at O (webpack-internal:///./node_modules/styled-components/dist/styled-components.browser.esm.js:31:19811)
    at ChangelogTrigger (webpack-internal:///./components/changelog/ChangelogTrigger.js:69:22)
```

Reply:
I've opened a new issue at, https://github.com/opencollective/opencollective/issues/5637. Will iterate tomorrow.